docs/development/doc_guide.rst

```diff
@@ -303,9 +303,9 @@ Markdown
 A few of PlasmaPy's files are written using Markdown_, such as README
 files and licenses from other packages. Markdown_ is simpler but more
 limited than reST_. Markdown_ files use the file extension :file:`.md`.
-Posts on GitHub are written in `GitHub Flavored Markdown`_. The
-following code block contains a few common examples of Markdown_
-formatting.
```

Comment:
Which reminds me I've been meaning to mention https://myst-parser.readthedocs.io/en/latest/index.html ...